src/System.Web.HttpUtility/tests/HttpUtility/HttpUtilityTest.cs

```diff
@@ -412,6 +412,7 @@ public void UrlDecode(string decoded, string encoded)
 
         [Theory]
         [MemberData(nameof(UrlDecodeData))]
+        [SkipOnTargetFramework(TargetFrameworkMonikers.NetFramework, "Full framework assumes the current codepage is UTF8")]
```

Comment:
> The inputs that are failing

We shouldn't disable the whole test if only two inputs are failing.  We should separate them out so that we can disable just the two that are failing.

> The inputs that are failing are the ones that have u0061 encoded which is correctly in UTF8

Am I misreading the test, or is there an explanation for why it fails when we're explicitly using UTF8 in the test?  Is this just a bug in the framework that we've fixed in core, or a difference between Unicode data sets being used, or something like that?